From c096cc0ca216af3c77cf10ba76a86b48f7eb5079 Mon Sep 17 00:00:00 2001
From: Andy CrossGate Yan <GeForce8800Ultra@gmail.com>
Date: Sat, 24 Sep 2022 11:40:52 +0000
Subject: [PATCH 3/3] vendor_lineage: Commonly build some packages

These were in handheld_product.mk and thus weren't included in GSI targets:
Contacts, DeskClock, Gallery2, fwb overlays

Change-Id: I00cc5918164ed70c612c5caa31eb8d2d0cd05b62
---
 config/common_mobile.mk | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/config/common_mobile.mk b/config/common_mobile.mk
index 1e3affa1..2f77827a 100644
--- a/config/common_mobile.mk
+++ b/config/common_mobile.mk
@@ -9,12 +9,16 @@ PRODUCT_PRODUCT_PROPERTIES += \
 # Apps
 PRODUCT_PACKAGES += \
     Backgrounds \
+    Contacts \
+    DeskClock \
     Eleven \
     Etar \
     ExactCalculator \
+    Gallery2 \
     Jelly \
     Profiles \
-    Seedvault
+    Seedvault \
+    frameworks-base-overlays
 
 ifneq ($(TARGET_EXCLUDES_AUDIOFX),true)
 PRODUCT_PACKAGES += \
-- 
2.25.1

